### ISSUE TITLE
Script deletes interfaces...

### ISSUE THREAD

**Issue by nathanringraham** [PNPtutorials/PNP_RPi3_AP#3]
If you don't know what you're doing, do not use this script. I could not use wifi or LAN after this (it deletes the interfaces). I had to completely re-install the OS onto my SD card.

**Comment by jrcharney** [comment]
I agree. I highly would NOT recommend running the ap.sh file if it deletes /etc/networks/interfaces unless you want to ruin everything INCLUDING the connection settings for eth0 which are also in this file and important!

**Comment by rajputanshulk** [comment]
backup your interfaces copy 1st
